.circleci/config.yml

```diff
@@ -377,7 +377,7 @@ jobs:
           command: .circleci/scripts/release-create-release-pr.sh
 
   prep-deps:
-    executor: node-browsers-small
+    executor: node-browsers-medium
```

Comment:
If we are bumping the executor size why are we then also changing the memory usage limits for `small`?

Reply:
Just changing the memory limit worked on some external PRs, but it couldn't get past a corrupt cache, so we had to go to `medium`.  I think @legobeat believes we should increase the memory usage limit anyway.

Reply:
I'm not against increasing the heap size for small machines, but it is a bit confusing to do it in this PR given that it _definitely_ doesn't help with the issue that this PR is for (which is the `prep-deps` step on CI runs from forks)

Reply:
@legobeat If you spot that OoM error, please let me know! I've been trying to keep an eye out for those lately.